django/db/backends/dummy/base.py

```diff
@@ -76,6 +76,7 @@ def __init__(self, *args, **kwargs):
         super(DatabaseWrapper, self).__init__(*args, **kwargs)
 
         self.features = BaseDatabaseFeatures(self)
+        self.features.supports_transactions = False
```

Comment:
I don't think so; base classes shouldn't depend on subclasses. Create a `DatabaseFeatures` subclass for the dummy backend and set the attribute there.